docs/data/data-grid/custom-columns/EditingWithDatePickers.js

```diff
@@ -18,131 +19,6 @@ import InputBase from '@mui/material/InputBase';
 import locale from 'date-fns/locale/en-US';
 import { styled } from '@mui/material/styles';
 
-function buildApplyDateFilterFn(filterItem, compareFn, showTime = false) {
```

Comment:
This is a positive side-effect of this PR - we can now reuse the `getGridDateOperators` for pickers integration instead of rewriting all filter operators 🙂